CMakeLists.txt

```diff
@@ -56,7 +56,10 @@ add_subdirectory(apps)
 add_subdirectory(tests)
 
 set(CPACK_PACKAGE_DESCRIPTION_FILE "${PROJECT_SOURCE_DIR}/README.md")
-set(CPACK_DEBIAN_PACKAGE_DEPENDS "libavahi-client3, avahi-daemon")
+set(SERVUS_PACKAGE_DEB_DEPENDS)
+if (avahi-client_FOUND)
+  list(APPEND SERVUS_PACKAGE_DEB_DEPENDS libavahi-client3 avahi-daemon)
+endif()
```

Comment:
That should be unconditional, otherwise catch-22

Reply:
I don't get your point. This variable is set for packaging *after* configuration. If avahi was not present at configuration time you don't want to add it a as package dependency.
This is not to be confused with SERVUS_DEB_DEPENDS, which lists these packages unconditionally.

Reply:
My bad. I confused it with DEB_PACKAGES.